Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8693,6 +8749,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the custom name visiblity of this monster. If it's false, you only see the name when you sight the mob. If it's true, you always see the custom name.",
 				},
+				SetLeashedTo =
+				{
+					Params =
+					{
+						{
+							Name = "Entity",
+							Type = "cEntity",
+						},
+					},
+					Notes = "Leash the monster to a entity.",
+				},
+				SetUnleashed =
+				{
+					Params =
+					{
+						{
+							Name = "ShouldDropPickup",
+							Type = "boolean",
+						},
+					},
+					Notes = "Unleash the monster from an entity.",
```

Comment:
At first I did what you are saying on your last proposal (put the hard work inside mob class) but that gave me some memory lock problems when closing the server.
So maybe creating those 2 new functions in cMonster will work. Sorry, but why not just export cEntity.AddLeashedMob / cEntity.RemoveLeashedMob / cEntity.HasAnyMobLeashed and hide cMonster's LeashTo / Unleash? They are fully functional for the purpose.